go.mod

```diff
@@ -62,14 +62,10 @@ require (
 
 require (
 	github.com/0xPolygon/go-ibft v0.0.0-20220810095021-e43142f8d267
-	go.uber.org/atomic v1.10.0
+	go.uber.org/atomic v1.10.0 // indirect
```

Comment:
Seems like you can entirely get rid of this reference. 

I've tried removing it manually from `go.mod` and running `go mod tidy` + build. Everything seems to be working fine.

Reply:
I've got the following result when running `go mod tidy`. Are you using go compiler version 1.17? Do you know any tips against that?

```bash
go mod tidy: go.mod file indicates go 1.18, but maximum supported version is 1.17
```

Reply:
@Stefan-Ethernal Just ping

Reply:
@Kourin1996 I'm running this version of Go:
`go version go1.18.4 linux/amd64`

If you are running go 1.17, I'd suggest updating installed version (I'm even puzzled how it was working for you before in that case).

Reply:
@Stefan-Ethernal Sorry, it was my mistake. Somehow I might have used go 1.17 at that time 🤦. Sorry for taking your time.
I updated go.mod and go.sum 091df51a34674dc53f4c05ba919db0a4983a4588